client/state/sites/updates/reducer.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SITE_UPDATES_RECEIVE,
+	SITE_UPDATES_REQUEST,
+	SITE_UPDATES_REQUEST_SUCCESS,
+	SITE_UPDATES_REQUEST_FAILURE,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+
+import { isValidStateWithSchema } from 'state/utils';
+import { itemsSchema } from './schema';
+
+export const items = ( state = {}, { siteId, type, updates } ) => {
```

Comment:
We can remove some boilerplate by using [createReducer](https://github.com/Automattic/wp-calypso/blob/master/client/state/utils.js#L57)
```javascript
export const items = createReducer( {}, {
    [SITE_UPDATES_RECEIVE]: ( state, action ) => return Object.assign( {}, state, { [ action.siteId ]: action.updates } 
} );
```